test/test_api_callable.py

```diff
@@ -42,6 +42,52 @@
 
 _DUMMY_ERROR_TYPE = face.AbortionError
 _DUMMY_ERROR_INSTANCE = _DUMMY_ERROR_TYPE(None, None, None, None)
+_CONFIG = {
+    'retry_codes_def': [
+        {
+            'name': 'foo_retry',
+            'retry_codes': ['code_a', 'code_b']
+        },
+        {
+            'name': 'bar_retry',
+            'retry_codes': ['code_c']
+        }],
+    'retry_params': [
+        {
+            'name': 'default',
```

Comment:
Are the types in the retry_params correct ?
Just checking, but I thought the YAML parser would not add quotes unless they were present in the yaml.